README.md

```diff
@@ -237,9 +237,9 @@ You see output specifying that TorchServe has stopped.
 
 
 ### Concurrency And Number of Workers
-TorchServe exposes configurations which allows the user to configure the number of worker threads on CPU and GPUs. This is an important config property that can speed up the server depending on the workload.
+TorchServe exposes configurations that allow the user to configure the number of worker threads on CPU and GPUs. There is an important config property that can speed up the server depending on the workload.
```

Comment:
TorchServe exposes configurations that allow the user to configure the number of worker threads on CPU and GPUs. This is an important config property that can speed up the server depending on the workload.